apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -1159,48 +1190,102 @@ t_leave_reestablish_(Cfg) ->
 
 leave_reestablish_(Cfg) ->
     Debug = get_debug(Cfg),
-    {I0, R0, Spec0} = channel_spec(Cfg),
-    #{ i := #{} = I
-     , r := #{} = R
+    {I0, R0, Spec0} = InitialSpec = channel_spec(Cfg),
+    ?LOG(Debug, "Initial spec = ~p", [InitialSpec]),
+    #{ i := #{} = I01
```

Comment:
Can you use sequential numbers here? So I0, I1, I2 ... instead of I0, I01

Reply:
I think it should be refactored into smaller functions instead.